.github/workflows/docs.yml

```diff
@@ -8,6 +8,10 @@ on:
   pull_request:
     paths:
     - 'docs/**'
+    # We now have a docs check that fails if any error codes don't have documentation,
+    # so it's important to do the docs build on all PRs touching the mypy/ directory
+    # in case somebody's adding a new error code without any docs
+    - 'mypy/**'  
```

Comment:
```suggestion
    # We now have a docs check that fails if any error codes don't have documentation,
    # so it's important to do the docs build on all PRs touching the mypy/ directory
    # in case somebody's adding a new error code without any docs
    - 'mypy/**'
```